src/promptflow/setup.py

```diff
@@ -54,6 +54,7 @@
             "promptflow-core[executor-service]",
             "promptflow-devkit[all]",
             "promptflow-azure",
+            "promptflow-parallel",
```

Comment:
put it here means that this is mandatory parts of promptflow.

Reply:
as discussed offline, it can be removed